src/main/kotlin/no/nav/arrangor/arrangor/ArrangorRepository.kt

```diff
@@ -84,7 +84,7 @@ class ArrangorRepository(
 	).firstOrNull()
 
 	data class ArrangorDbo(
-		val id: UUID = UUID.randomUUID(),
```

Comment:
Dette har jeg tatt bort for at det skal være mer synlig hvor vi setter en ny id, siden det er noe vi ikke egentlig skal gjøre ennå (der det gjøres nå brukes kun fra apier som ikke er i bruk ennå)